tests/containers/base_image_test.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+import logging
+import pathlib
+import tempfile
+from typing import TYPE_CHECKING
+
+import testcontainers.core.container
+import testcontainers.core.waiting_utils
+
+from tests.containers import docker_utils
+
+logging.basicConfig(level=logging.DEBUG)
+LOGGER = logging.getLogger(__name__)
+
+if TYPE_CHECKING:
+    import pytest_subtests
+
+
+class TestBaseImage:
+    """Tests that are applicable for all images we have in this repository."""
+
+    def test_oc_command_runs(self, image: str):
+        container = testcontainers.core.container.DockerContainer(image=image, user=123456, group_add=[0])
+        container.with_command("/bin/sh -c 'sleep infinity'")
+        try:
+            container.start()
+            ecode, output = container.exec(["/bin/sh", "-c", "oc version"])
+        finally:
+            docker_utils.NotebookContainer(container).stop(timeout=0)
+
+        logging.debug(output.decode())
+        assert ecode == 0
+
+    def test_oc_command_runs_fake_fips(self, image: str, subtests: pytest_subtests.SubTests):
```

Comment:
@jstourac suggested skipping this test when running suite on openshift, openshift will test real configurations, not these fake ones

Reply:
* https://github.com/opendatahub-io/notebooks/issues/848